perfdash/Makefile

```diff
@@ -1,6 +1,7 @@
 all: push
 
 # See deployment.yaml for the version currently running-- bump this ahead before rebuilding!
+# Additionally please ensure to update the same in www/VERSION
```

Comment:
Could you add a static analysis that validate that VERSION file contents are equal the TAG in make file?

Reply:
Added a check before we build the container. 
Thanks for the review!